resources/definitions/gmax15plus_dual.def.json

```diff
@@ -11,13 +11,13 @@
         "file_formats": "text/x-gcode",
         "platform": "gmax_1-5_xt-plus_s3d_full model_150707.stl",
 		"has_variants": true,
-        "has_machine_quality": true,
 		"variants_name": "Hotend",
-		"preferred_variant_name": "0.5mm E3D (Default)",
+		"preferred_variant_name": "*0.5mm E3D (Default)*",
```

Comment:
Remove `*`, we don't use wildcards for this setting. Otherwise when adding a new gMax printer to Cura, the variant will be empty.